l3kernel/l3keys.dtx

```diff
@@ -1813,12 +1813,17 @@
 % \end{macro}
 %
 % \begin{macro}{\@@_property_find:n}
-% \begin{macro}
+% \begin{macro}[EXP]
 %   {
 %     \@@_property_find_auxi:w   ,
+%     \@@_property_find_auxiii:w
```

Comment:
`\@@_property_find_auxiv:w` is missing from this list (if we want to be this precise here).